Library/Homebrew/rubocops/lines.rb

```diff
@@ -381,6 +381,14 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
 
             problem "Use the `#{match}` Ruby method instead of `#{method.source}`"
           end
+
+          # Enforce use of OpenMPI for MPI dependency in core
+          find_method_with_args(body_node, :depends_on, "mpich") do |method_node|
+            next unless formula_tap == "homebrew-core"
```

Comment:
I'm guessing it's because of this. We are not specifying the tap in the tests, are we?